src/java.base/share/conf/security/java.security

```diff
@@ -157,9 +157,35 @@ RestrictedSecurity.NSS.140-2.securerandom.algorithm = PKCS11
 #endif
 
 #if defined aix-ppc || defined linux-ppc || defined linux-s390 || defined linux-x86 || defined windows
+#
+# Strict RestrictedSecurity mode profile for FIPS 140-3. This policy represents only allowable
+# approved cryptography in the OpenJCEPlusFIPS provider along with other non cryptographic algorithms
+# commonly used. Please note that the following constraints represent algorithms no longer allowable
+# within this strict profile for the OpenJCEPlusFIPS provider as compared to the
+# "RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Weaklyenforced" profile defined below.
+#    {MessageDigest,MD5,*}, \
+#    {MessageDigest,SHA-1,*}, \
+#    {MessageDigest,SHA-512/224,*}, \
+#    {Signature,SHA1withRSA,*}, \
+#    {Signature,RSAforSSL,*}, \
+#    {Signature,NONEwithDSA,*}, \
+#    {Signature,NONEwithECDSA,*}, \
```

Comment:
Both here in this comment and in the actual property, there should be a space after each comma to improve readability, and the constraints should be sorted alphabetically.

Reply:
Done